runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -118,6 +118,7 @@
 	private final Provider<ClientThread> clientThreadProvider;
 	private final CardLayout cardLayout = new CardLayout();
 	private final Rectangle sidebarButtonPosition = new Rectangle();
+	private final Rectangle sidebarButtonRange = new Rectangle();
```

Comment:
This field is no longer necessary too

Reply:
That depends on whether we want to check if the mouse is in the area of the button (pretty small if you don't know where it is) or if the mouse is in a larger "sidebarButtonRange" area, which is currently set to the right-side of the client, which is easily readable and configurable as a Rectangle object...